src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -0,0 +1,141 @@
+        if extra_query_kwargs is not None:
+            assert 'environment' not in extra_query_kwargs
+            query_kwargs.update(extra_query_kwargs)
+
+        query_kwargs['environments'] = environments if environments else None
+        result = search.query(**query_kwargs)
+        return result, query_kwargs
+
+    def get(self, request, organization):
+        """
+        List an Organization's Issues
+        `````````````````````````````
+
+        Return a list of issues (groups) bound to an organization.  All parameters are
+        supplied as query string parameters.
+
+        A default query of ``is:unresolved`` is applied. To return results
+        with other statuses send an new query value (i.e. ``?query=`` for all
+        results).
+
+        The ``statsPeriod`` parameter can be used to select the timeline
+        stats which should be present. Possible values are: '' (disable),
+        '24h', '14d'
+
+        :qparam string statsPeriod: an optional stat period (can be one of
+                                    ``"24h"``, ``"14d"``, and ``""``).
+        :qparam bool shortIdLookup: if this is set to true then short IDs are
+                                    looked up by this function as well.  This
+                                    can cause the return value of the function
+                                    to return an event issue of a different
+                                    project which is why this is an opt-in.
+                                    Set to `1` to enable.
+        :qparam querystring query: an optional Sentry structured search
+                                   query.  If not provided an implied
+                                   ``"is:unresolved"`` is assumed.)
+        :pparam string organization_slug: the slug of the organization the
+                                          issues belong to.
+        :auth: required
+        """
+        stats_period = request.GET.get('statsPeriod')
+        if stats_period not in (None, '', '24h', '14d'):
+            return Response({"detail": ERR_INVALID_STATS_PERIOD}, status=400)
+        elif stats_period is None:
+            # default
+            stats_period = '24h'
+        elif stats_period == '':
+            # disable stats
+            stats_period = None
+
+        environments = list(Environment.objects.filter(
+            organization_id=organization.id,
+            name__in=self.get_environments(request, organization),
+        ))
+
+        serializer = functools.partial(
+            StreamGroupSerializerSnuba,
+            environment_ids=[env.id for env in environments],
+            stats_period=stats_period,
+        )
+
+        query = request.GET.get('query', '').strip()
+        if query:
+            # check to see if we've got an event ID
+            if len(query) == 32:
+                groups = list(
+                    Group.objects.filter_by_event_id(
```

Comment:
this probably needs to be updated to respect the environment filter too, but might file a ticket and follow up on that